.github/workflows/pr.yaml

```diff
@@ -0,0 +1,43 @@
+name: pr
+on: [ pull_request ]
```

Comment:
shouldn't this be `[ push ]` to compile on any new commit?

Reply:
It updates on each push but online if there's an open pr